src/runtime/methodbinder.cs

```diff
@@ -415,11 +416,32 @@ internal Binding Bind(IntPtr inst, IntPtr args, IntPtr kw, MethodBase info, Meth
                 }
                 else
                 {
-                    op = (arrayStart == paramIndex)
-                        // map remaining Python arguments to a tuple since
-                        // the managed function accepts it - hopefully :]
-                        ? Runtime.PyTuple_GetSlice(args, arrayStart, pyArgCount)
-                        : Runtime.PyTuple_GetItem(args, paramIndex);
+                    if(arrayStart == paramIndex)
+                    {
+                        // for a params method, we may have a sequence or single/multiple items
+                        // here we look to see if the item at the paramIndex is there or not
+                        // and then if it is a sequence itself.
+                        IntPtr item = Runtime.PyTuple_GetItem(args, paramIndex);
+                        Runtime.PyErr_Clear();
```

Comment:
That would just eat all exceptions. Should add proper handling.

Reply:
Ok, I will revisit.

Reply:
I removed this completely based up updates.